ebscopy/ebscopy.py

```diff
@@ -376,15 +375,15 @@ def request(self, method, data, session_token=None, attempt=0):
 
 		try:
 			headers['x-authenticationToken']	= self.auth_token
-		except:
-			if method is not "UIDAuth":
-				raise ValueError("Missing Authentication Token!")
+		except ValueError as e:
+			if method != "UIDAuth":
```

Comment:
Is this correct?
https://realpython.com/python-is-identity-vs-equality/

Reply:
No, it appears not :)